### PR TITLE
bump to 4.18 and new 4.17 branch config

### PR DESCRIPTION
**Pull request by rauhersu**

### CODE REVIEW

**Comment by rauhersu** [comment]
Once we merge this, we must merge the following upstream ztp changes:

4.17: https://github.com/openshift-kni/cnf-features-deploy/pull/2012
master: https://github.com/openshift-kni/cnf-features-deploy/pull/2013

/cc @nishant-parekh

File: core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay [hunk 1]
```diff
@@ -86,7 +86,8 @@ registry.ci.openshift.org/ocp-kni/4.15:ztp-site-generator quay.io/openshift-kni/
 registry.ci.openshift.org/ocp-kni/4.16:cnf-tests quay.io/openshift-kni/cnf-tests:4.16 quay.io/openshift-kni/cnf-tests:4.16.0 quay.io/openshift-kni/cnf-tests:latest
 registry.ci.openshift.org/ocp-kni/4.16:dpdk quay.io/openshift-kni/dpdk:4.16 quay.io/openshift-kni/dpdk:4.16.0 quay.io/openshift-kni/dpdk:latest
 registry.ci.openshift.org/ocp-kni/4.16:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.16 quay.io/openshift-kni/ztp-site-generator:4.16.0
-registry.ci.openshift.org/ocp-kni/4.17:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.17 quay.io/openshift-kni/ztp-site-generator:4.17.0 quay.io/openshift-kni/ztp-site-generator:latest
+registry.ci.openshift.org/ocp-kni/4.17:ztp-site-generator quay.io/openshift-kni/ztp-site-generator:4.17 quay.io/openshift-kni/ztp-site-generator:4.17.0
```
Comment (nishant-parekh) [inline core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:89]: @rauhersu I think we need to bump the other images in this file as well or at least notify the folks to do this..

**Review by nishant-parekh — APPROVED** [review]
/lgtm 
/approve
@rauhersu one small comment to follow up with cnf-tests and dpdk owners to bump the image mapping for them too..

**Comment by lack** [comment]
I think the `e2e-aws-ci-tests` always fails, so those results look "good enough" to me.

**Comment by rauhersu** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-e2e-aws-ci-tests

**Comment by yuvalk** [comment]
came to approve, but I see there's an open comment
https://github.com/openshift/release/pull/56058/files#r1735119445

File: core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay (hunk 1 above)
Comment (nishant-parekh) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:89]: @yuvalk @rauhersu is going to notify the necessary folks to make the requested updates here once this change merges.. we discussed this.

Comment (nishant-parekh) [reply core-services/image-mirroring/openshift-kni/mapping_openshift-kni_quay:89]: we can consider this comment resolved for the PR merge perspective..

**Comment by yuvalk** [comment]
/approve

FYI @SchSeba and @sshnaidm

**Comment by rauhersu** [comment]
/pj-rehearse pull-ci-openshift-kni-cnf-features-deploy-master-e2e-aws-ci-tests

**Pull request merged**